### ISSUE TITLE
Very important Ability to select multiple layers with the move cursor

### ISSUE THREAD

**Issue by gybeasmz** [photopea/photopea#748]
This is a basic function that belongs to the absolute basics of photo editing softwares. However it will not be possible to implemenet unless you resolve the layer locking issue. See the default state when you open a file in other editors (like PS for example) is, that the background layer (the first white layer) is locked, That allows you to drag a rectangle across it with the move cursor. It´s the same principle as when you want to select multiple icons on your desctop. You just draw a rectangle. Same has to be implemented in any good photo editor. You should be able to draw rectangle with the move tool and all the layers withing that rectangle get selected. Imagine how much time this saves. But unless there is the ability to lock layers + plus by default the background layer is locked , it will not be possible to draw the rectangle, because you would move the background. This is another dealbreaker.

**Comment by photopea** [comment]
Interesting. I never used this feature in any editor. 

But what about holding Shift and clicking on each object? Isn't it the same?

**Comment by gybeasmz** [comment]
Imagine that you have a wall and you have to paint it different color every 30 minutes. You have a pretty big pointed brush, but instead of making a couple of long strokes strokes from left to right and up and down to paint the wall in 3 minutes, you decide to point the brush perpendicularly to the wall and spend the whole 30 minutes by making hundreds and hundreds little dots with the tip of that brush to paint that wall. 30 minutes are gone, but guess what ... you gotta start all over again. Give it one day and you will get insane and just burn the place down. See when you are wireframing or even designing a website let´s say, there can be hundreds of layers layers, texts, rectangles  and you have to move them around all the time, sometimes many times until you get it right. You need to have a tool that allows you to select groups of layers immediately.

**Comment by photopea** [comment]
That is what Layers panel is for ... if you have a nice structure, you just click on the folder of layers, and move the whole folder :D :P But I see, that modern designers do not use Layers panel at all, so we have to add this feature for them.

**Comment by MichalSZZ** [comment]
I use that function in Illustrator but never in Photoshop (I work by clicking objects with shift button on artboard or on layer panel). But I think, it should work like that:
![selecting-objects](https://user-images.githubusercontent.com/38118273/41104451-aa8502b8-6a6b-11e8-9dcf-fbf411c9df17.gif)

**Comment by gybeasmz** [comment]
For me and I´m positive that for big percentage of users, this would be one of the top features missing.

**Comment by photopea** [comment]
Hi, it was implemented. The locked layers are not selectable this way.

I am very sorry for the three month delay :(

**Comment by SephReed** [comment]
Wait, so how do I move a folder of layers?

**Comment by photopea** [comment]
You can select a folder in the Layers panel.

**Comment by SephReed** [comment]
> You can select a folder in the Layers panel.

Yes, but then when I click on the canvas it selects an item in that folder.  I've found I can use arrow keys to move it around though, which works well enough.  Still, it would be nice to click a folder in the layers panel, then click something on the canvas and be able to move it.  Even if I had to click shift, or click somewhere strange in the folders panel to indicate I don't want to select children by clicking on them.
